core/src/test/scala/com/yahoo/maha/core/query/hive/HiveQueryGeneratorV2Test.scala

```diff
@@ -1930,12 +1932,14 @@ class HiveQueryGeneratorV2Test extends BaseHiveQueryGeneratorTest {
                           "cube": "s_stats",
                           "selectFields": [
                               {"field": "Advertiser ID"},
-                              {"field": "CTR"}
```

Comment:
Can this test change go into a distinct test?  We want to ensure existing tests go unharmed at worst, and improved at best.

Reply:
pushed a new commit to separate the test